archspec/cli.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import archspec
+import archspec.cpu
+import argparse
+import sys
+
+
+def main():
+    parser = argparse.ArgumentParser(description="archspec command line interface", prog='archspec')
+
+    parser.add_argument('--cpu-host', help="Print name of host CPU microarchitecture", action='store_true')
+    parser.add_argument('--version', action='version', version=archspec.__version__)
```

Comment:
@alalazo `archspec/__init__.py` currently hardcodes `__version__` to `'0.1.0'`, but the version is also defined in `pyproject.toml`.

Is there a way to avoid hardcoding the version in two places (only in `pyproject.toml`)?